### ISSUE TITLE
Script to move from old browser labeling to the new scheme

### ISSUE THREAD

**Issue by karlcow** [webcompat/webcompat.com#2532]
This will run only once. It should be done **AFTER** #2531

It will reparse all issues and creates a recategorization for labels:

* [ ] browser-[name] or browser-fixme for unknown browsers
* [ ] os-{windows,macos,linux,android,ios}
* [ ] form-{mobile,desktop,tablet}
* [ ] version-nnn

This will probably require a massive analysis offline for all the bugs. 
Using https://github.com/josegonzalez/python-github-backup would be a good idea to create a table of all issues and their labels.

Some things to keep in mind.

* The current labeling is probably more exact than the ones guessed from parsing the issue body.
* The creation of new labels should probably not erased right away the old ones (in case of catastrophe).
* A timer should be put in place so we don't get flagged by GitHub. (Maybe an early warning email could help)

**Comment by laghee** [comment]
Hey, @karlcow -- since I've run the backup this week and am working on sorting it into a local database of current issues/labels/milestones/events, do let me know if I can help out here in any way.

**Comment by karlcow** [comment]
@laghee yes if you want you can start coding a script that will browse this backup and prepare all the data to be modified leaving out the function that will edit live the labels so we do not do anything accidently. 

Once #2531 is done we would just have to run the script. :)

**Comment by karlcow** [comment]
oh and… I'm pretty sure by creating the script will find… plenty of issues :D

**Comment by laghee** [comment]
Short term simplified sql database of bare bones, mostly current bugs and labels data for anyone interested to play with (should work easily with `pg_restore`): https://send.firefox.com/download/a5faac998d/#cw7pkom5zu6yXD3bVIEX3w

Tables:
- issues (id & open status)
- labels (id & name)
- issue_labels (issue_id & label_id)

**Comment by karlcow** [comment]
@laghee https://screenshots.firefox.com/CfuTX9bcWy66dXfT/send.firefox.com

**Comment by karlcow** [comment]
> (I don't think I can double check this right now because I no longer have access to the Heroku account.)

hmmm… :) How can we solve this?
